ts/data.ts

```diff
@@ -1,6 +1,25 @@
 const data = {
   view: 'entry-form',
-  entries: [],
+  entries: [] as FormEntry[],
   editing: null,
   nextEntryId: 1,
 };
```

Comment:
Rather than creating this object directly, you should be calling `readData` and let that return the object you want to assign